recipe/meta.yaml

```diff
@@ -27,42 +27,96 @@ source:
     - nvcc.profile.patch.win  # [win]
 
 build:
-  number: 1
+  number: 2
   skip: true  # [osx or win]
 
-requirements:
-  build:
-    - {{ compiler("c") }}
-    - {{ compiler("cxx") }}
-    - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - sysroot_{{ target_platform }} 2.17    # [linux]
-  host:
-    - cuda-version {{ cuda_version }}
-  run:
-    - {{ pin_compatible("cuda-version", max_pin="x.x") }}
-    - cuda-cudart-dev
-  run_constrained:
-    - gcc_impl_{{ target_platform }} >=6,<13  # [linux]
-    - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - cmake >=3.20
+outputs:
+  - name: cuda-nvcc
+    files:
+      - bin                                               # [linux]
+      - targets/{{ target_name }}/bin                     # [linux]
+      - targets/{{ target_name }}/nvvm                    # [linux]
```

Comment:
```suggestion
```

Reply:
These were in the targets package in the first iteration of this PR, here's the [failure lines](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=693157&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=274)

I suspect it has to do with this being a `noarch: generic` package some incompatibility with the `libgcc-ng` being used.

Also, targets packages (at least from a cross compilation perspective) typically only provide the unversioned `.so` and `.h` files. So I moved `targets/../{bin,nvvm}` back to `cuda-nvcc` as I'm not sure they should be in the `noarch` package. These should only be installed on the host architecture AFAIK.

Reply:
> These should only be installed on the host architecture AFAIK.

The way I anticipate these being used is that a recipe will say

```yaml
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}
```

and this will be installed on the host architecture. If on x86-64 and cross-compiling for ARM, `compiler('cuda')` should install `cuda-nvcc_linux64`. Right?

Reply:
In the simple non-cross case, `compiler('cuda')` should enable a user to call `nvcc`. That means the `$PREFIX/bin` must be installed with `cuda-nvcc_linux64` as I understand it.

Reply:
Yeah that sounds right

Would suggest we try this change and then look at the results in CI together so we can debug it